### DISCUSSION TITLE
Welcome to winstall Discussions!

### DISCUSSION THREAD

**Discussion by MehediH** [omaha-consulting/winstall#46] [General]
## 👋 Welcome!
  Going to be using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about one feature that you think would be useful for winstall!

**Comment by mdyunusraza** [comment]
Hi,
Came across this wonderful program of yours. I would like to know how can i host the website internally in my organization and use it for installing the apps. I could not find any documentation of how to use the files in the project to host it in our own web server and server internally.

**Comment by ezequielsousabr** [comment]
O winstall não tem suporte para versões mais antigas como windows 7, 8 ?

**Comment by mrobirasm** [comment]
Hello,

I can't create any pack because it gives error when I try to login with twitter.

It may be because of the latest changes that Twitter is undergoing.
